packages/cli/commands/feedback.js

```diff
@@ -0,0 +1,34 @@
+const open = require('open');
+
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const {
+  FEEDBACK_OPTIONS,
+  FEEDBACK_URLS,
+} = require('@hubspot/cli-lib/lib/constants');
+const { logger } = require('@hubspot/cli-lib/logger');
+
+const {
+  feedbackTypePrompt,
+  bugPrompt,
+  generalPrompt,
+} = require('../lib/prompts/feedbackPrompt');
+
+const i18nKey = 'cli.commands.project.subcommands.feedback';
+
+exports.command = 'feedback';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async () => {
```

Comment:
One thing we could do is additionally support flags in this command (may be worth double checking this w/ design though).

It could work something like this:
`hs feedback --bug` automatically opens our GH issues repo
`hs feedback` does what you currently have coded up (prompts for which type)